get_info.py

```diff
@@ -1,28 +1,168 @@
 import requests
 import json
+import sys
 from secrets import returnYelpApi
 
 from yelp.client import Client
 
 import get_info
 
-
 api_key = returnYelpApi()
 headers = {'Authorization': 'Bearer %s' % api_key}
 
 def jprint(obj):
     text = json.dumps(obj, sort_keys=True, indent=4)
     print(text)
 
-def main():
+def generateCategories(categoriesList):
+    returnArray = []
+    for c in categoriesList:
+        returnArray.append(c['title'])
+    return returnArray
+
+def putAllInDb(input, postlink):
+    businessList = input['businesses']
+    listlen = len(businessList)
+    for b in businessList:
+
+        notCredible = False
+
+        if 'name' in b:
+            Name = b['name']
+
+        if 'id' in b:
+            id = b['id']
+
+        Address = "No address listed..."
+        if 'location' in b:
+            address = b['location']['display_address']
+            if len(address) >= 2:
+                Address = address[0] + '\n' + address[1]
+            else:
+                notCredible = True
```

Comment:
maybe just continue here instead of going through all the code and then skipping?